common/src/main/java/io/novafoundation/nova/common/base/BaseActivity.kt

```diff
@@ -44,15 +41,13 @@ abstract class BaseActivity<T : BaseViewModel> :
         inject()
         initViews()
         subscribe(viewModel)
-    }
-
-    abstract fun inject()
 
-    abstract fun layoutResource(): Int
+        viewModel.errorLiveData.observeEvent(::showError)
 
-    abstract fun initViews()
+        viewModel.messageLiveData.observeEvent(::showMessage)
```

Comment:
Why do we now subscribe in `attachBaseContext`?

Reply:
No, we subscribe in onCreate

Reply:
It seems that the problem is in how github displays changes